vertx-web/src/test/java/io/vertx/ext/web/RouterTest.java

```diff
@@ -978,15 +978,15 @@ public void testConsumesCTParamsIgnored() throws Exception {
   @Test
   public void testConsumesNoContentType() throws Exception {
     router.route().consumes("text/html").handler(rc -> rc.response().end());
-    testRequest(HttpMethod.GET, "/foo", 404, "Not Found");
+    testRequest(HttpMethod.GET, "/foo", 415, "Unsupported Media Type");
```

Comment:
Would it be interested to say "No media type specified" (that could help client) ? But maybe that's too specific (and maybe not standard ?)